### ISSUE TITLE
Create card for `covr_coverage`

### ISSUE THREAD

**Issue by AARON-CLARK** [pharmaR/riskassessment#533]
I think once PR #529 is merged, we'll have to include a new card for the `covr_coverage` metric. We've kind of been ignoring it for a long time since's its almost always `NA`. just adding a "Percentage card" similar to bug closure rate (seen below) with updated labels should do the trick. Since we'll soon be migrating to one tab for all `riskmetric` metrics, I think it belongs on the "maintenance metrics" tab for now.

![image](https://github.com/pharmaR/riskassessment/assets/17878953/382d94cd-c8c4-4c03-953a-6a59c6e32aad)

_Originally posted by @AARON-CLARK in https://github.com/pharmaR/riskassessment/pull/529#pullrequestreview-1462770012_

**Comment by Jeff-Thompson12** [comment]
Closed with #545
